process/securing_your_workspace.md

```diff
@@ -0,0 +1,15 @@
+# Securing Your Workspace
```

Comment:
Other items worth noting in this doc: 
 * Physical security for the office: what's expected?
 * How about 'after the project is over'. Any special concerns there?  
 * How to escalate when something goes wrong.